.github/workflows/hide_bot_comments.yml

```diff
@@ -0,0 +1,24 @@
+name: Hide Bot Comments
+on:
+  push:
+    branches: [master]
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [master]
+
+jobs:
+  hide-bot-comments:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
```

Comment:
Is there a reason for not using v2?

Reply:
I didn't know about v2. is there an easy way to check for the latest version?

Reply:
🤔 I usually have to check the repo...

You can setup dependabot to upgrade them automatically https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates